src/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -20,6 +22,10 @@
     <ProjectJson>unix/project.json</ProjectJson>
     <ProjectLockJson>unix/project.lock.json</ProjectLockJson>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(TargetGroup)' == 'net46' and '$(ProjectJson)' == '' ">
+    <ProjectJson>net46/project.json</ProjectJson>
```

Comment:
The only cases where you would need a new project.json file is if you have multiple configurations that need different dependencies but under the same TFM. So your windows vs unix example if they both target netstandard14 but need different dependencies then we need to split into multiple project.json files because we cannot have unique dependencies based on RID. However that isn't the case for net46 so you can just add that section into your existing default project.json file.